setup.py

```diff
@@ -41,6 +41,7 @@ def get_file_text(file_name):
     install_requires=[
         "urllib3>=1.26.11",
         "certifi",
+        "opentelemetry-distro>=0.35b0",  # XXX check lower bound
```

Comment:
let's reconsider this separately before shipping, we really don't want to force the otel dependency on errors-only users, so this needs some thought.
Basically, if people want to use performance, we need to force them to do `pip install sentry-sdk[opentelemetry]`,

Reply:
Wrote down something at https://www.notion.so/sentry/Errors-Only-Experience-After-POTel-e7208959efb144a9b647ac2c494b09c2?pvs=4